src/components/number-input/_number-input.scss

```diff
@@ -155,8 +155,14 @@
     top: 2.25rem;
   }
 
-  .#{$prefix}--number--helpertext:not(.#{$prefix}--number--nolabel) .#{$prefix}--number__controls {
-    top: 3.625rem;
+  .#{$prefix}--number--helpertext:not([data-invalid]) .#{$prefix}--number__controls {
+    top: unset;
```

Comment:
`unset` is not support in IE11 - Probably `auto`?

Reply:
@asudoh updated 😄